deploy/crds/zookeeper_v1beta1_zookeepercluster_cr.yaml

```diff
@@ -7,8 +7,9 @@ spec:
   image:
     repository: pravega/zookeeper
     tag: 0.2.8
+  storageType: persistence
   persistence:
-    reclaimPolicy: Delete
+    reclaimPolicy: Retain
```

Comment:
Why are we changing this? This has implications for Pravega and we need it to be Delete for Pravega to work on reinstall without reinstalling Zookeeper.

Reply:
I have created two sample cr for both pravega and ecs and mentioned in the cr folder